src/SelectSingle.vue

```diff
@@ -216,14 +235,13 @@
 			onOptionChange(index: number) {
 				this.activeIndex = index
 			},
+			optionExists(option: SelectOption) {
```

Comment:
Do we need this method? Since it invokes and returns another helper function, could we just cut out the middle man, so to speak?

Reply:
It might over-complicate things! It makes it a *little* easier to write when being used (we're always going to have the same `this.uniqueIdField` when in the component, so we don't need to pass it in each time), but I think you're right - let's just add the helper to the instance & call it directly